### ISSUE TITLE
No longer functioning

### ISSUE THREAD

**Issue by Sikksens** [nicholastay/node-twitch-get-stream#9]
Any update to this with new Twitch v5 API?

**Comment by nicholastay** [comment]
Are you sure that it is no longer working - perhaps check your client ID? Seems to still be just fine on my end.

**Comment by d-fischer** [comment]
Interestingly, I checked the API you're using yesterday, and it was just fine - but now I get this:
```json
{
    "error": "Gone",
    "message": "this API has been removed.",
    "status": 410
}
```
You're gonna have to find another API that does what you need :(

**Comment by dkonopka** [comment]
It looks like https://usher.ttvnw.net/api/channel/hls/ is no longer available :(

**Comment by nicholastay** [comment]
Strange, because I tested it again just now and it still works. I even opened the network devtools when opening up a stream in a browser and seems that the same /access-token -> usher/.../.m3u8 flow is still being used.

I could also still watch the stream in VLC just fine.

**Comment by dkonopka** [comment]
Yup, it's weird this endpoint API works again properly.

**Comment by roxaloxa** [comment]
I'm also getting status 410 on mine, no longer working :(

**Comment by jamezrin** [comment]
Twitch does something really strange, if your client id has been using the v5 API a lot in the past, you can use its endpoints without a problem, but if you use a recently created one, you will get this response
```
{
  "error": "Gone",
  "status": 410,
  "message": "this API has been removed."
}
```

Or that is what I understood, I might be wrong though. I need to debug more.

**Comment by jamezrin** [comment]
Basically, just use this client id `kimne78kx3ncx6brgo4mv6wki5h1ko`.
Any 3rd party client id gets the `410 Gone` error.

**Comment by roxaloxa** [comment]
Surely doing this is a one-way ticket to Twitch nuking that client ID? 🤔

**Comment by jamezrin** [comment]
If there were the case, wouldn't it be gone already since youtube-dl and streamlink use it?

**Comment by roxaloxa** [comment]
Good enough of a point for me! I'd be terrified if I had any stake in this not working but it's just a personal script so I'll go ahead and use it. 😛

**Comment by nicholastay** [comment]
Should be published on npm as 0.6.0 - sorry for delays. If anything else changes, I guess we should use youtube-dl as a good reference for future.

**Comment by galipmedia** [comment]
kimne78kx3ncx6brgo4mv6wki5h1ko is no longer working for me does anyone have an alternative solution?
